rest_specification/restful_service_specification.md

```diff
@@ -83,9 +83,21 @@ for `user` if listing all users is disallowed for performance reasons.
 
 ## Operations Against Specific Resources
 Specific resources will be addressed through the base URI plus the identifier
-for that resource.
+for that resource by one or more field names, as many as required to identify
+a single entity.
 
-    /user/{id}
+    /user/v1?id={id}
```

Comment:
To be clear the "v1" is not new: it was already in the spec just not in all examples. Sorry for updating multiple things at once.